lsp/lsp-harness/Cargo.toml

```diff
@@ -7,11 +7,6 @@ edition = "2021"
 anyhow = "1.0.71"
 env_logger = "0.10.0"
 log = "0.4.19"
-lsp-types = "0.94.0"
+lsp-types = "0.88.0"
```

Comment:
Any reason for a downgrade? We might update NLS to work with the latest version as well, if it's trivial. But maybe you prefer to do that in a subsequent PR

Reply:
I had to make it compatible with the version in the nls crate. I originally tried bumping the version in nls, but it's incompatible with `codespan-lsp`. (BTW, codespan doesn't seem actively maintained, so we may need to do something about that)